aerotech_101smc2/aerotech_101smc2/_aerotech.py

```diff
@@ -39,7 +40,7 @@ def get_state(self):
         return state
 
     def _set_position(self, position):
-        self._serial_port.write(f"M {position}\n".encode())
+        self._serial_port.write(f"M {position}\n".encode())        
```

Comment:
try not to include trailing whitespace (just a style issue)

I use `pre-commit` and `black` to help me ensure that the codestyle is correct---you might wish to do the same